src/amo/components/SectionLinks/styles.scss

```diff
@@ -16,11 +16,16 @@
   color: $grey-40;
   display: inline-block;
   font-weight: 400;
-  margin: 0 12px;
+  margin: 0 8px;
   outline: none;
+  overflow-wrap: break-word;
```

Comment:
I suggest we add a comment to explain why `break-word` is needed here, given that we should only rely on the global/default `overflow-wrap: anywhere` rule in this project.